pyproject.toml

```diff
@@ -1,3 +1,96 @@
+[tool.poetry]
+name = "epyqlib"
+version = "0.0.0"
+description = ""
+authors = ["Alex Anker <alex.anker@epcpower.com>"]
+
+[tool.poetry-dynamic-versioning]
+#modifies epyqlib/__init__.py
+enable = true
+vcs = "git"
+style = "pep440"
+
+[tool.poetry.dependencies]
+python = "3.7.9"
+
+#### EPC Repos ####
+sunspecdemo = {git = "https://github.com/epcpower/sunspec-demo", rev = "6a07576a9d3c16f00a31121ff96edcbfe0d73822"}
+###################
+
+alqtendpy = "0.0.4"
+arrow = "0.12.1"
+appdirs = "1.4.3"
+attrs = ">=19.3.0"
+bitstruct = "6.0.0"
+boto3 = "1.14.41.0"
+boto3-type-annotations = "0.3.1"
+canmatrix = "0.9.1"
+certifi = "2020.6.20"
+certitude = { version = "1.0.1", markers = "sys_platform == 'Darwin'" }
+click = "<=7.1.2"
+dulwich = "0.20.6"
+fab = "3.0.0"
+GitPython = "2.1.15"
+graham = "0.1.11"
+marshmallow = "2.16.3"
+natsort = "5.5.0"
+paho-mqtt = "1.4.0"
+Pint = "0.11"
+pyelftools = "0.26"
+PyQt5 = ">=5.13.0"
+##pyqt5-tools fails
+##pyqt5-tools = { version = "5.14.1.3", markers = "sys_platform == 'win32'" }
+qt5reactor = "0.5"
+python-dateutil = "2.7.5"
+python-docx = "0.8.7"
+python-dotenv = "0.9.1"
+QtAwesome = "0.6.0"
+siphash-cffi = "^0.1.4"
+treq = "21.1.0"
+Twisted = "21.2.0"
+twine = "1.13.0"
+PyQt5Designer = { version = "^5.14.1", markers = "sys_platform == 'win32'" }
+python-can = "4.0.0.dev0"
+pysunspec = "2.1.0"
+
+[tool.poetry.dev-dependencies]
+poetry-dynamic-versioning = "^0.13.1"
+black = "20.8b1"
+pytest = "5.3.5"
+pytest-qt = "3.3.0"
+pytest-rerunfailures = "5.0"
+pytest-twisted = "<=1.13.4"
+pytest-xvfb = "1.2.0"
+
+[tool.poetry.scripts]
+collectdevices = 'epyqlib.collectdevices:main'
```

Comment:
Is there a reason the `outparsecheck` script from `setup.py` was not put on this list?
`outparsecheck = epyqlib.outparsecheck:main [dss]`

Reply:
ill add it but i dont think we actually use it.